### PR TITLE
Add SGN V2 delegations to celer-sgn-delegation [celer-sgn-delegation]

### PR DESCRIPTION
**Pull request by Dominator008**
Changes proposed in this pull request:
- Count the delegations to Celer SGN V2 Staking

### CODE REVIEW

File: src/strategies/celer-sgn-delegation/index.ts [hunk 1]
```diff
@@ -86,49 +88,81 @@ const abi = [
...
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/celer-sgn-delegation/index.ts:116]:
```suggestion
const v2StakingABI = [
  'function getDelegatorTokens(address _delAddr) view returns (uint256, uint256)'
];
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/celer-sgn-delegation/index.ts [hunk 2]
```diff
@@ -167,11 +201,38 @@ export async function strategy(
...
+      v2StakingABI,
+      addresses.map((address: string) => [
+        options.v2StakingViewerAddress,
+        'getDelegatorTokens',
+        [address]
+      ]),
+      { blockTag }
+    )
+  ).map((value: string[]) => value[0]);
+  const v2DelegatorTokensMap = addresses.reduce((map, address, i, _) => {
```
Comment (ChaituVR) [inline src/strategies/celer-sgn-delegation/index.ts:220]:
```suggestion
  const v2DelegatorTokensMap = addresses.reduce((map, address, i) => {
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/celer-sgn-delegation/index.ts (hunk 2 above)
Comment (Dominator008) [reply src/strategies/celer-sgn-delegation/index.ts]: Done.

File: src/strategies/celer-sgn-delegation/index.ts (hunk 1 above)
Comment (Dominator008) [reply src/strategies/celer-sgn-delegation/index.ts]: Done.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**